interface/.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Do we need any of files inside .idea in our main code repo ?

Reply:
I think that got added by freya in a commit, but it should just be config files for IntelliJ

Reply:
Don't think its needed other than her local machine